app/views/shared/_navbar.html.erb

```diff
@@ -2,41 +2,41 @@
   <div class="container-fluid">
     <ul class="navbar-nav me-auto">
       <li class="nav-item active">
-        <%= link_to "Home", root_path, class: "nav-link" %>
+        <%= link_to "Venue Finder", root_path, class: "nav-link" %>
       </li>
     </ul>
-      <%# <%= image_tag "https://thenounproject.com/api/private/icons/2228292/edit/?backgroundShape=SQUARE&backgroundShapeColor=%23000000&backgroundShapeOpacity=0&exportSize=752&flipX=false&flipY=false&foregroundColor=%23000000&foregroundOpacity=1&imageFormat=png&rotation=0&token=gAAAAABj8Qy6DrlH_tS72zuG6RRtijaY0FFooZL9gXu1NJ4qu3qzhLgvUFKT_VKbdFmm4LU1iQZpvyGCrUk2NTtWQM-9XTzlDA%3D%3D" %>
-    <%# <% end %>
 
     <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarSupportedContent" aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
       <span class="navbar-toggler-icon"></span>
     </button>
 
     <div class="collapse navbar-collapse" id="navbarSupportedContent">
       <ul class="navbar-nav me-auto">
-        <li class="nav-item">
-          <%= link_to "All Venues", venues_path, class: "nav-link" %>
```

Comment:
Hey Jan, all looks good. My only question on this would be on moving all venues behind a login. It's just a choice of design but I'd normally expect to be able to see an index of available venues prior to logging in.

Reply:
Hi, agree. We got action button on banner on home page for Venus which will lead you to index (/venues) and you don`t have to log in. In picture there is still All venues showed as I haven`t merged Navbard PR yet. 

![image](https://user-images.githubusercontent.com/105860641/221158105-5a496bcd-2178-40c9-a6f9-d9a1797e9724.png)